Lib/sysconfig/__init__.py

```diff
@@ -507,6 +510,9 @@ def _init_config_vars():
         import _osx_support
         _osx_support.customize_config_vars(_CONFIG_VARS)
 
+    if sys.platform in {'ios', 'tvos', 'watchos'}:
+        _CONFIG_VARS['installed_base'] = sys.executable
+
```

Comment:
Is this still necessary now that we're using the standard POSIX scheme? If so, it would be useful to include a comment explaining what assumptions it's making about the filesystem layout.

Reply:
Looks like you're right - this isn't needed any more either.